src/app/static/src/app/store/baseline/actions.ts

```diff
@@ -21,10 +22,22 @@ export const actions: ActionTree<BaselineState, RootState> & BaselineActions = {
             .postAndReturn<PjnzResponse>("/baseline/pjnz/", formData);
     },
 
+    async uploadShape({commit}, formData) {
+        await api<BaselineActionTypes, BaselineErrorActionTypes>(commit)
+            .withSuccess("ShapeUploaded")
+            .withError("ShapeUploadError")
+            .postAndReturn<PjnzResponse>("/baseline/shape/", formData);
+    },
+
     async getBaselineData({commit}) {
```

Comment:
Could fire these off simultaneously rather than waiting for pjnz to complete before requesting shape?

Reply:
Great point